azavea/tilegarden
# Feature/pug demo

## Overview

This PR replaces the static HTML demos with those build from Pug templates, with the intent that the demos will be easier to maintain and add to in the future. HTML files are built on merge to `gh-pages`. Demo page source files now live in `src/pug/` and are built using a Docker container.

## Testing Instructions

 * Note that there is no longer a `demo/` directory in the repository.
 * Run `./scripts/update && ./scripts/build-demo-pages` to generate demo pages, in the places in the repository they were formerly.


Resolves #82
